metpy/calc/tests/test_tools.py

```diff
@@ -14,7 +14,8 @@
                         nearest_intersection_idx, pressure_to_height_std,
                         reduce_point_density, resample_nn_1d, second_derivative)
 from metpy.calc.tools import (_get_bound_pressure_height, _greater_or_close, _less_or_close,
-                              _next_non_masked_element, delete_masked_points)
+                              _next_non_masked_element, delete_masked_points, parse_angle,
```

Comment:
In general, our calculation functions are all grabbed from `metpy.calc`, and the module they live inside is just an implementation detail. Instead of importing `parse_angle` directly from `metpy.calc.tools`, you should mark the function as exported (see below), and just import it from `metpy.calc`. This way the test checks that the function is properly exported from `metpy.calc`.